keras_core/ops/numpy_test.py

```diff
@@ -285,6 +285,79 @@ def test_shape_equal_ignore_axes(self):
         self.assertTrue(knp.shape_equal(x, y, axis=(1, 2)))
         self.assertFalse(knp.shape_equal(x, y, axis=1))
 
+    def test_shape_equal_only_none(self):
+        x = KerasTensor([None, None]).shape
+        y = KerasTensor([5, 6]).shape
+        self.assertTrue(knp.shape_equal(x, y, allow_none=True))
+
+    def test_shape_equal_axis_as_list(self):
+        x = KerasTensor([3, 4, 5]).shape
+        y = KerasTensor([3, 6, 5]).shape
+        self.assertTrue(knp.shape_equal(x, y, axis=[1]))
+
+    def test_shape_non_equal_with_negative_axis(self):
+        x = KerasTensor([3, 4, 5]).shape
+        y = KerasTensor([3, 4, 6]).shape
+        self.assertFalse(knp.shape_equal(x, y, axis=-2))
+
+    def test_shape_equal_with_negative_axis(self):
+        x = KerasTensor([3, 4, 5]).shape
+        y = KerasTensor([3, 4, 5]).shape
+        self.assertTrue(knp.shape_equal(x, y, axis=-1))
+
+    def test_shape_equal_zeros(self):
+        x = KerasTensor([0, 4]).shape
+        y = KerasTensor([0, 4]).shape
+        self.assertTrue(knp.shape_equal(x, y))
+        y = KerasTensor([0, 5]).shape
+        self.assertFalse(knp.shape_equal(x, y))
+
+    def Test_Broadcast_Shapes_conversion_to_list():
```

Comment:
Please use snake case for all test methods, e.g. `test_broadcast_shapes_conversion_to_list(self)`

Also note the need for `self`.

Reply:
Done